content/webapp/services/prismic/types/exhibition-guides.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  PrismicDocument,
+  RichTextField,
+  RelationField,
+  GroupField,
+  NumberField,
+  LinkToMediaField,
+} from '@prismicio/types';
+import { Image } from '.';
+
+export type ExhibitionLink = PrismicDocument<{
+  title: RichTextField;
+  // TODO: properly type the promo from linked exhibition
+  promo: never;
+  description: RichTextField;
+}>;
+
+export type ExhibitionGuidePrismicDocument = PrismicDocument<{
+  title: RichTextField;
+  relatedExhibition: RelationField<'exhibitions'>;
+  components: GroupField<{
+    number: NumberField;
+    title: RichTextField;
+    tombstone: RichTextField;
+    image: Image;
+    description: RichTextField;
+    audioWithDescription: LinkToMediaField;
+    audioWithoutDescription: LinkToMediaField;
+    bslVideo: LinkToMediaField;
```

Comment:
I'd thought this was more likely to be a YouTube embed, rather than something we hosted ourselves?

Reply:
Oh yes, very good point!